features/proxy_config.feature

```diff
@@ -42,23 +37,20 @@ Feature: Proxy configuration
         esm-infra     +yes      +disabled      +UA Infra: Extended Security Maintenance \(ESM\)
         """
         When I run `truncate -s 0 /var/log/squid/access.log` `with sudo` on the `proxy` machine
-        When I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
+        And I run `ua config set ua_apt_https_proxy=http://<ci-proxy-ip>:3128` with sudo
+        Then stdout matches regexp:
         """
-        contract_url: 'https://contracts.canonical.com'
-        data_dir: /var/lib/ubuntu-advantage
-        log_level: debug
-        log_file: /var/log/ubuntu-advantage.log
-        ua_config:
-          apt_http_proxy: http://<ci-proxy-ip>:3128
-          apt_https_proxy: http://<ci-proxy-ip>:3128
+        Setting APT proxy
+        Setting proxy on ua scope
```

Comment:
I think it would be more clear if this was one line that said `Setting UA-scoped APT proxy`